Cython/Compiler/Builtin.py

```diff
@@ -226,7 +226,23 @@ def declare_in_type(self, self_type):
     #('reduce',    "",     "",      ""),
     BuiltinFunction('reload',     "O",    "O",     "PyImport_ReloadModule"),
     BuiltinFunction('repr',       "O",    "O",     "PyObject_Repr", builtin_return_type='str'),
-    #('round',     "",     "",      ""),
+    BuiltinFunction('round',      "f",   "f",    "__Pyx_round_float",
+                    utility_code=UtilityCode.load("round_float", "Builtins.c"),
+                    is_strict_signature=True, nogil=True),
+    BuiltinFunction('round',      "d",   "d",    "__Pyx_round_double",
+                    utility_code=UtilityCode.load("round_double", "Builtins.c"),
+                    is_strict_signature=True, nogil=True),
+    BuiltinFunction('round',      None,   None,    "__Pyx_round_longdouble",
+                    utility_code=UtilityCode.load("round_longdouble", "Builtins.c"),
+                    func_type=PyrexTypes.CFuncType(
+                        PyrexTypes.c_longdouble_type, [
+                            PyrexTypes.CFuncTypeArg("arg", PyrexTypes.c_longdouble_type, None)
+                            ],
+                        is_strict_signature=True, nogil=True)),
+    BuiltinFunction('round',      "O",   "O",    "__Pyx_PyFloat_round",
+                    utility_code=UtilityCode.load("PyFloat_round", "Builtins.c")),
+    BuiltinFunction('round',      "OO",   "O",    "__Pyx_PyFloat_round2",
+                    utility_code=UtilityCode.load("PyFloat_round2", "Builtins.c")),
```

Comment:
> I assume it was necessary to add these?

It was. And I do agree it was a degradation since previously they could share __pyx_builtin_round, while now they have to get it every time.